std-bits/table/src/main/java/org/enso/table/expressions/ExpressionVisitorImpl.java

```diff
@@ -59,38 +60,47 @@ public interface MethodInterface {
     Object[] prepareArguments(Value[] args, Function<Object, Value> makeConstantColumn);
   }
 
+  public record EnsoType(Value module, Value type, boolean isStaticMethod) {
+    public EnsoType(String moduleName, String typeName, boolean isStaticMethod) {
+      this(
+          Context.getCurrent().getBindings("enso").invokeMember("get_module", moduleName),
+          Context.getCurrent()
+              .getBindings("enso")
+              .invokeMember("get_module", moduleName)
+              .invokeMember("get_type", typeName),
+              isStaticMethod);
+    }
```

Comment:
You seem to `Context.getCurrent().getBindings("enso").invokeMember("get_module", moduleName)` twice. You can avoid that by having another _layer_ of constructors:
```suggestion
      this(
          Context.getCurrent().getBindings("enso").invokeMember("get_module", moduleName),
          typeName, isStaticMethod);
    }
    
    private EnsoType(Value module, String typeName, boolean isStaticMethod) {
      this(module, module.invokeMember("get_type", typeName), isStaticMethod);
    }
```

Reply:
Done